www/%team/distributing/index.html.spt

```diff
@@ -1,10 +1,19 @@
 # encoding: utf8
+
+from aspen import Response
 from gratipay.utils import get_team
 
 [-----------------------------------------------------------------------------]
 team = get_team(state)
 if team.available == 0:
     website.redirect('..', base_url='')
+
+if user.participant == None:
+    raise Response(401)
+
+if not (user.ADMIN or user.participant.member_of(team) or user.participant.username == team.owner):
+    raise Response(403)
+
```

Comment:
> I was thinking that since there were no link to this page for the ordinary user, it was not supposed to be reachable.

It's supposed to be reachable, this is just another case where we're only making this available for Team Gratipay right now. Actually, we should still make it generally available (the Team Gratipay distributing page). Maybe we can handle this and Receiving in a new PR (cf. https://github.com/gratipay/gratipay.com/pull/3873#issuecomment-164165342)?